### ISSUE TITLE
Add API to list downloaded files

### ISSUE THREAD

**Issue by rrhartjr** [aerokube/selenoid#791]
Similar to the /download single file API it would be helpful to have an API to list all downloaded files, in case it is not known ahead of time what the downloaded file would be

**Comment by vania-pooh** [comment]
@rrhartjr duplicate of https://github.com/aerokube/selenoid-images/issues/173.

**Comment by vania-pooh** [comment]
@rrhartjr the main problem is that this change will require rebuilding images.

**Comment by vania-pooh** [comment]
@rrhartjr fixed. Currently I have no time to rebuild the image - you have to build:
1) `selenoid/base:5.0` from [sources](https://github.com/aerokube/selenoid-images/tree/master/selenium/base/base_5) using `docker build`.
2) Then build image with e.g. Chrome using instructions: https://github.com/aerokube/selenoid-images/tree/master/selenium/base/base_5

**Comment by rrhartjr** [comment]
TL;DR: It works! Thanks!

I have two questions:
- Can you get a JSON list?
- Can I increase the timeout of the sessions? It removes them sometimes while I am debugging...

More info for anyone interested in building images --

I ran this command to build the new image: `sh automate_chrome.sh 76.0.3809.132-1 76.0.3809.126 76.0-rrhartjr`

I went here to find the newest chrome version: https://www.ubuntuupdates.org/package/google_chrome/stable/main/base/google-chrome-stable
I went here to find the applicable chrome driver version: https://chromedriver.chromium.org

These images show up now
```
$ docker images
REPOSITORY                    TAG                    IMAGE ID            CREATED              SIZE
selenoid/chrome               76.0-rrhartjr          47a6947f44d4        About a minute ago   883MB
selenoid/vnc                  chrome_76.0-rrhartjr   47a6947f44d4        About a minute ago   883MB
selenoid/vnc_chrome           76.0-rrhartjr          47a6947f44d4        About a minute ago   883MB
```

I changed my selenoid `browsers.json` to this
```
{
	"chrome": {
		"default": "rrhartjr",
		"versions": {
			"rrhartjr": {
				"image": "selenoid/chrome:76.0-rrhartjr",
				"port": "4444",
				"tmpfs": {"/tmp":"size=512m"}
			}
		}
	}
}
```

Ran a simple WebDriver script `webDriver.get("https://the-internet.herokuapp.com/download/Sample.png");`

Ran this curl command and it returned a list of the downloaded files
```
$ curl http://localhost:4444/download/72c218fed0edabbadc772e6c101e10d7/
<pre>
<a href="Sample.png">Sample.png</a>
</pre>
```

**Comment by vania-pooh** [comment]
@rrhartjr I think should be `curl http://localhost:4444/download/cb08821fa7e7b8d2e9a5c69a2cd9d39c/?json` (with a slash before `?json`).

**Comment by rrhartjr** [comment]
Still the same result, HTML list
```
$ curl http://localhost:4444/download/268860c05ac77ff44a2b450f4c08c3c4/?json
<pre>
<a href="Sample.png">Sample.png</a>
</pre>
```

**Comment by rrhartjr** [comment]
This appears to be because the browser Dockerfile is using the `selenoid/base:5.0` version, which predates the new fileserver support

```
$ head chrome/apt/Dockerfile
FROM selenoid/base:5.0

ARG VERSION
ARG CLEANUP
```

I tried to rebuild the base_5 image, using: `docker build . -t selenoid/base:rrhartjr` from the `selenium/base/base_5` directory, but got this error:

```
# _/fileserver
main_test.go:4:2: cannot find package "github.com/aandryashin/matchers" in any of:
	/usr/local/go/src/github.com/aandryashin/matchers (from $GOROOT)
	/go/src/github.com/aandryashin/m
```

I commented out `go test` from the base_5 Dockerfile and it built.

Then I ran the container, started the fileserver and it returns json now

```
$ docker run -it 8ae8f5ceb1a7
root@3d8c96a913f3:/# /usr/bin/fileserver &
[1] 10
root@3d8c96a913f3:/static# touch /static/test.txt
root@3d8c96a913f3:/static# curl http://localhost:8080?json
["test.txt"]
root@3d8c96a913f3:/static# curl http://localhost:8080
<pre>
<a href="test.txt">test.txt</a>
</pre>
["test.txt"]
```

I don't know enough about Docker to know what or if anything needs to be done...

**Comment by vania-pooh** [comment]
@rrhartjr yes, I mentioned in my previous comments that you have to rebuild this base image.

**Comment by rrhartjr** [comment]
Oops my mistake! Thanks for the help!

**Comment by Durgam777** [comment]
@vania-pooh and team
Can you please help in unzip the file on selenoid and get the extracted files list?

If any help from anyone would be great.
Thanks,

**Comment by vania-pooh** [comment]
@Mypaal7 I don't thing this is Selenoid responsibility.

**Comment by Mypaal7** [comment]
@vania-pooh 
ok, Thank you

**Comment by sudarsunperu** [comment]
@vania-pooh and team, 
I have an use case that requires to know the list of downloaded files. Should I build the images as discussed above or the API is already part of any releases?
Thank you

**Comment by vania-pooh** [comment]
@sudarsunperu should be available in latest browser versions.
